aspnetcore/includes/net-core-prereqs-windows.md

```diff
@@ -1,3 +1,6 @@
-[Visual Studio for Windows](https://www.microsoft.com/net/download/windows)
-* **ASP.NET and web development** workload
-* **.NET Core cross-platform development** workload
+[Visual Studio for Windows](https://www.microsoft.com/net/download/windows).
+Select the **ASP.NET and web development** workload.
+
+::: moniker range=">= aspnetcore-2.1"
+[.Net Core 2.1 SDK](https://github.com/aspnet/Home/wiki/2.1.0-Early-Access-Downloads)
```

Comment:
We shouldn't point people to the early access download for 2.1 from the docs. We should just wait for the RTW release and point to that.